doc: add security-steward rotation information (#41707)
nodejs/node: doc/contributing/security-release-process.md

LIKE THIS**_ which will be replaced during the release process with the
information described.

## Security release stewards

For each security release, a security steward will take ownership for
coordinating the steps outlined in this process. Security stewards
are nominated through an issue in the TSC repository and approved
through the regular TSC consensus process. Once approved, they
are given access to all of the resources needed to carry out the
steps listed in the process as outlined in
[security steward on/off boarding](security-steward-on-off-boarding.md).

The current security stewards are documented in the main Node.js
[README.md](https://github.com/nodejs/node#security-release-stewards).

| Company    | Person   | Release Date |
| ---------- | -------- | ------------ |

> | NearForm   | Matteo   | 2021-Oct-21  |
> | Datadog    | Bryan    | 2022-Jan-22  |

Reviewer: These dates are incorrect. 
```suggestion
| NearForm   | Matteo   | 2021-Oct-12  |
| Datadog    | Bryan    | 2022-Jan-10  |
```
https://nodejs.org/en/blog/vulnerability/jan-2022-security-releases/
https://nodejs.org/en/blog/vulnerability/oct-2021-security-releases/

Reviewer 2: This confusion is why this should have used this format from the start. :-D

LIKE THIS**_ which will be replaced during the release process with the
information described.

## Security release stewards

For each security release, a security steward will take ownership for
coordinating the steps outlined in this process. Security stewards
are nominated through an issue in the TSC repository and approved

> through the regular TSC consensus process. Once approved they

Reviewer 2: ```suggestion
through the regular TSC consensus process. Once approved, they
```

> ## Security release stewards
> 
> For each security release a Security steward will take ownership for
> co-ordinating the steps outlined in this process. Security stewards
> are nominated through an issue in the TSC repository and approved
> through the regular TSC concensus process. Once approved they
> are given access to all of the resources needed to carry out the
> steps listed in the process as outined in
> [security steward on/off boarding](security-steward-on-off-boarding.md).

Reviewer 3: ```suggestion
## Security release stewards

For each security release, a security steward will take ownership for
coordinating the steps outlined in this process. Security stewards
are nominated through an issue in the TSC repository and approved
through the regular TSC consensus process. Once approved they
are given access to all of the resources needed to carry out the
steps listed in the process as outlined in
[security steward on/off boarding](security-steward-on-off-boarding.md).
```

The current security stewards are documented in the main Node.js

> [README.md](https://github.com/nodejs/node#security-stewards).

Reviewer 2: ```suggestion
[README.md](https://github.com/nodejs/node#security-release-stewards).
```

> The currently rotation is as follows:

Author: ```suggestions
The current rotation is as follows:
```

> The current rotation is as follows:
> 

Reviewer 2: I don't think you need to explain that it's the rotation. Just show the table.

```suggestion
```

| Company    | Person   | Release Date |
| ---------- | -------- | ------------ |
| NearForm   | Matteo   | Oct 21       |

> | Data Dog   | Bryan    | Jan 22       |

Reviewer 4: ```suggestion
| Datadog    | Bryan    | Jan 10       |
```

Reviewer 4: Err, just the name. I had thought the number referred to the date, but it refers to the year.

> | NearForm   | Matteo   | Oct 21       |
> | Datadog    | Bryan    | Jan 22       |

Reviewer 2: Include the year?

```suggestion
| NearForm   | Matteo   | 2021-Oct-21  |
| Datadog    | Bryan    | 2022-Jan-22  |
```

Reviewer: I think confusingly the number referred to the year -- the most recently January release went out well before 22 Jan 2022.

| RH and IBM | Joe      |              |
| NearForm   | Matteo   |              |

> | Data Dog   | Vladimir |              |

Reviewer 5: ```suggestion
| Datadog   | Vladimir |              |
```

Reviewer 5: (per @bengl's suggestion)

> | Datadog   | Vladimir |              |

Reviewer 4: This row is now one space off, but no big deal since it should render fine.

Author: but the linter will not like it, fixing.

> | Datadog   | Vladimir |              |

Author: ```suggestion
| Datadog    | Vladimir |              |
```
